build.csx

```diff
@@ -0,0 +1,212 @@
+#load "packages/simple-targets-csx.5.2.0/simple-targets.csx"
+
+using static SimpleTargets;
+
+var  solutionName = "FakeItEasy";
+
+var solution = "./" + solutionName + ".sln";
+var packagesDirectory = Path.GetFullPath("packages");
+var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+var versionAssembly = "./src/FakeItEasy/bin/Release/FakeItEasy.dll";
+var nuspecs = Directory.GetFiles("./src", "*.nuspec");
+var gitlinks = Directory.GetDirectories("./src").Where(p => !p.EndsWith(".Shared")).Select(Path.GetFileName);
+
+var unitTests = new []
+{
+  "tests/FakeItEasy.Tests/bin/Release/FakeItEasy.Tests.dll",

```

Comment:
😱 The indent on this line (and on all the other array initialiser argument lines) is 2 spaces instead of 4!